src/commands/weather.js

```diff
@@ -112,8 +117,8 @@ class Weather extends Aquarius.Command {
       case '19':   // Dust
       case '20':   // Foggy
       case '21':   // Haze
-      case '22':   // Smoky
         return EMOJI_LIST.FOGGY;
+      case '22':   // Smoky
```

Comment:
I think for Smoky the foggy emoji makes more sense than the windy one - swap this and I'll merge